examples/app.py

```diff
@@ -24,6 +24,20 @@
 
 r"""Minimal Flask application example for development.
 
+Install the Invenio default theme
+
+You should execute these commands in the examples-directory.
+
+.. code-block:: console
+   $ pip install invenio-theme
+   $ pip install invenio-assets
+   $ flask -a app.py npm
+   $ cd static
+   $ npm install
```

Comment:
Please make sure that the example app is tested. You might need to update `.travis.yml` and add `tests/test_example_app.py`.

--
See changes in https://github.com/inveniosoftware/invenio-marc21/commit/bc0c8928c3f7a44f7e7ead97000623e26555e6f4